here-naksha-lib-core/src/test/java/com/here/naksha/lib/core/models/storage/ReadFeaturesTest.java

```diff
@@ -9,17 +9,18 @@
 
 public class ReadFeaturesTest {
 
-  @Test
-  void testShallowCopy() throws JsonProcessingException {
-    // given
-    ReadFeatures readFeatures = new ReadFeatures();
-    Json jsonGenerator = Json.get();
-
-    // when
-    String json = jsonGenerator.writer().writeValueAsString(readFeatures);
-
-    // then
-    String expectedJson = "{\"collectionIds\":[],\"limit\":100000,\"limitVersions\":1,\"noFeature\":false,\"noGeoRef\":false,\"noGeometry\":false,\"noMeta\":false,\"noTags\":false,\"queryDeleted\":false,\"queryHistory\":false,\"resultFilter\":[],\"returnHandle\":false}";
-    assertEquals(expectedJson, json, "there is a property change in ReadFeatures, add it to shallowCopy and update json");
-  }
+//  @Test
+//  void testShallowCopy() throws JsonProcessingException {
+//  TODO uncomment when ReadRequest is support in PgSession
+//    // given
+//    ReadFeatures readFeatures = new ReadFeatures();
+//    Json jsonGenerator = Json.get();
+//
+//    // when
+//    String json = jsonGenerator.writer().writeValueAsString(readFeatures);
+//
+//    // then
+//    String expectedJson = "{\"collectionIds\":[],\"limit\":100000,\"limitVersions\":1,\"noFeature\":false,\"noGeoRef\":false,\"noGeometry\":false,\"noMeta\":false,\"noTags\":false,\"queryDeleted\":false,\"queryHistory\":false,\"resultFilter\":[],\"returnHandle\":false}";
+//    assertEquals(expectedJson, json, "there is a property change in ReadFeatures, add it to shallowCopy and update json");
+//  }
```

Comment:
Does it have to be commented?
If you know how - try to fix this plz, otherwise add a TODO comment

Reply:
I will add a prefix, the TODO is already on line 14